cmd/vulcan-prowler/main.go

```diff
@@ -70,12 +70,14 @@ var (
 			Check the Details and Resources sections to know the compliance status
 			and more details.
 		</p>`,
+		Labels: []string{"compliance", "cis"},
```

Comment:
I would suggest to add `aws` tag in this and following `Labels` entries.
As better tagged are findings better filtering we can apply later.
However, it is true that you can infer that is an AWS resource from the target/assettype.

Reply:
You are totally right, thx u :)

It's done now!